srcpkgs/kops/template

```diff
@@ -18,8 +18,10 @@ do_fetch() {
 	# process expects the directory to be a git repository
 	rm -rf $wrksrc
 	mkdir -p $wrksrc/src/k8s.io
-	git clone -b ${version} https://github.com/kubernetes/kops \
+	git clone https://github.com/kubernetes/kops \
 		$wrksrc/src/k8s.io/kops
+	cd $wrksrc/src/k8s.io/kops
+	git checkout v${version}
```

Comment:
I think this would work with `-b v${version}`. `git-clone(1)` claims `-b` can take tags, not only branch names.

Reply:
Thank you for the suggestion, I have updated the PR.